src/test/kotlin/blackjack/model/CardTest.kt

```diff
@@ -0,0 +1,13 @@
+package blackjack.model
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class CardTest {
+    @Test
+    fun `should create a card with given rank and suit`() {
+        val card = Card(Rank.TWO, Suit.HEARTS)
+        assertThat(card.rank.toString()).isEqualTo("TWO")
+        assertThat(card.suit.toString()).isEqualTo("HEARTS")
+    }
+}
```

Comment:
```suggestion
    @Test
    fun `should create a card with given rank and suit`() {
        val card = Card(Rank.TWO, Suit.HEARTS)
        assertThat(card.rank).isEqualTo(Rank.TWO)
        assertThat(card.suit).isEqualTo(Suit.HEARTS)
    }
}
```

Although it might seem not important, I think it would be better to use the enum itself
when asserting the contents inside a card.

I recommend using `toString()` only for logging or debugging purposes,
rather than as making an output, as part of a logic, or as test validation value.

Well, I guess `TWO` and `HEARTS` could count as debugging / logging info as well. 😂

Reply:
Thanks for the insightful suggestion! I agree that using the enum values directly in assertions is clearer and less fragile than relying on `toString()`. I have updated the tests to compare enums directly.